packages/appstore/src/components/cfds-listing-logged-out/cfds-listing-logged-out.tsx

```diff
@@ -48,16 +50,17 @@ const CFDsListingLoggedOut = observer(() => {
                     />
                 );
             })}
-            {!is_eu_user && (
-                <div className='cfds-listing-logged-out__cfd-full-row'>
-                    <hr className='cfds-listing-logged-out__divider' />
-                </div>
-            )}
-            {!is_eu_user && (
-                <div className='cfds-listing-logged-out__cfd-full-row'>
-                    <Text weight='bold'>{localize('Deriv cTrader')}</Text>
-                </div>
+            {available_ctrader_accounts.length > 0 && (
+                <React.Fragment>
+                    <div className='cfds-listing-logged-out__cfd-full-row'>
+                        <hr className='cfds-listing-logged-out__divider' />
+                    </div>
+                    <div className='cfds-listing-logged-out__cfd-full-row'>
+                        <Text weight='bold'>{localize('Deriv cTrader')}</Text>
```

Comment:
brand names don't need to be localized

Reply:
It was already localized. I just move it inside condition